x-pack/plugins/ml/public/application/settings/filter_lists/components/add_item_popover/add_item_popover.js

```diff
@@ -112,7 +112,11 @@ export class AddItemPopover extends Component {
                 />
               }
             >
-              <EuiTextArea value={this.state.itemsText} onChange={this.onItemsTextChange} />
+              <EuiTextArea
+                value={this.state.itemsText}
+                onChange={this.onItemsTextChange}
+                data-test-subj={'mlAddFilterListItemTextArea'}
```

Comment:
To be consistent with other naming, I'd suggest to make this `mlFilterListAddItemTextArea`.